eZ/Publish/API/Repository/ContentService.php

```diff
@@ -305,12 +305,14 @@ public function deleteVersion(VersionInfo $versionInfo);
      * Loads all versions for the given content.
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to list versions
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException if invalid status
```

Comment:
Please type-hint API exception. I'd also rephrase description to use verb, e.g. `if the given status is invalid`